src/controllers/expenses.js

```diff
@@ -0,0 +1,110 @@
+'use strict';
+
+const expensesServices = require('../services/expenses');
+
+const getAll = (req, res) => {
+  const expenses = expensesServices.filterExpenses(req.query);
+
+  res.send(expenses);
+};
+
+const getOne = (req, res) => {
+  const { id } = req.params;
+  const foundExpenses = expensesServices.getById(id);
+
+  if (!foundExpenses) {
+    res.status(404).send('Expenses not found');
+
+    return;
+  }
+
+  res.send(foundExpenses);
+};
+
+const add = (req, res) => {
+  const {
+    userId,
+    spentAt,
+    title,
+    amount,
+    category,
+    note,
+  } = req.body;
+
+  const isRequiredField = !spentAt
+    || !title
+    || !amount
+    || !category
+    || !note;
+
+  if (isRequiredField) {
+    res.status(400).send('One of the field is required');
+
+    return;
+  }
+
+  const newExpense = expensesServices.create(
+    userId,
+    spentAt,
+    title,
+    amount,
+    category,
+    note,
+  );
+
+  res.status(201).send(newExpense);
+};
+
+const remove = (req, res) => {
+  const { id } = req.params;
+  const foundExpenses = expensesServices.getById(id);
+
+  if (!foundExpenses) {
+    res.status(404).send('Expenses not found');
```

Comment:
Consider adding a more descriptive error message, e.g. 'Expense not found' instead of 'Expenses not found'